_posts/newsletters/2022-01-31-newsletter.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: US-RSE Newsletter
+subtitle: January 2022 Newsletter
+category: newsletter
+tags: [newsletter, survey, sc22, scipy22]
```

Comment:
I'm wondering about these tags - do they or will they have relevance outside of this post? E.g., mostly sc22 and scipy22. If we had a tag for scipy maybe that would make sense, but _just_ scipy22? The tag space is going to get very big (and hard to navigate) if we have tags for every conference, every year. maybe every conference instead?

Reply:
I debated this very thing actually. I had it as scipy and then thought that we never write it as just SC, so to be consistent with SC, I put the 22 on scipy. But I have no strong opinion here.

Reply:
My preference would be for the general tags, so they have meaning more than once, and then naturally in the search the latest will be at the top. It doesn't have a ton of meaning to just have for one year. If sc is too small, maybe just supercomputing instead?

Reply:
ok, went with supercomputing and scipy